Paws4Me/Paws4Me/Controllers/SignInViewController.swift

```diff
@@ -6,53 +6,96 @@
 //
 import UIKit
 
-class SignInViewController: UIViewController {
+protocol Validation {
+    func isValidCredentials(username: String, password: String) -> Bool
+}
 
+protocol borderCustomize {
+    func addUnderline(_ textField1: UITextField, _ textField2: UITextField, bottomLine: CALayer, bottomLine2: CALayer)
+}
+
+class SignInViewController: UIViewController, borderCustomize, Validation {
     @IBOutlet weak var txtUsername: UITextField!
     @IBOutlet weak var txtPassword: UITextField!
     @IBOutlet weak var btnSignIn: UIButton!
     @IBOutlet weak var btnRegister: UIButton!
-    
     var bLoggedIn = false
```

Comment:
Naming can also be improved here: 

`var isLoggedIn = false`

Reply:
Thank you for the suggestion 👍🏻